cpp/json_ffi/json_ffi_engine.cc

```diff
@@ -78,13 +78,19 @@ bool JSONFFIEngine::AddRequest(std::string request_json_str, std::string request
     }
   }
 
+  bool pinned_system_prompt = false;
```

Comment:
we should pass in debug_config into serve::DebugConfig directly, instead of pas sby per value